test/server-unit/numberToText.spec.js

```diff
@@ -0,0 +1,78 @@
+const { expect } = require('chai');
+const numberToText = require('../../server/lib/NumberToText');
+const en = require('../../client/src/i18n/en/numbers.json');
+const fr = require('../../client/src/i18n/fr/numbers.json');
+const util = require('../../server/lib/util');
+const _ = require('lodash');
+
+const loadDictionary = _.clone(util.loadDictionary);
```

Comment:
I don't think you need to clone this.  Just replace it like:
```js
const loadDictionary = util.loadDictionary; 
```

It looks like lodash doesn't support cloning functions.